pgnReader.py

```diff
@@ -0,0 +1,38 @@
+'''
+    path - path to .pgn file
+    maxGame - maximum number of games to parse
+'''
+def pgnParse(path,maxGame=None):

```

Comment:
what about returning not a list, but an **object** of a class you've created? For example, you may have a Game class, with moves, and other metadata (player ELOs) - and this class can have even its own methods, e.g. converting to different representations?

Reply:
Do you mean a list of Game-class objects in which each object is one game, or storing all games in one object?

Reply:
That depends if you have some non-typical behavior you want to have in the ListOfGames class. If not (a normal Python list suffices because you only need appending etc.) then there is no need to have one and just return a plain list of Game objects.

BTW, you may use Python typing to show that this function returns `typing.List[Game]`